docs/_process.md

```diff
@@ -0,0 +1,8 @@
+## Design and Build Process
+### How we create and build components.
+
+1. Prototype
+2. Sketch.cloud
+3. Add to theme repo (fractal)
+4. PR
```

Comment:
I think there should be a step in the process to include the relevant react/vue components as Clay suggested in code review.